cloudify_cli/tests/commands/test_blueprints.py

```diff
@@ -64,6 +80,14 @@ def test_blueprints_delete(self):
         self.client.blueprints.delete = MagicMock()
         self.invoke('blueprints delete a-blueprint-id')
 
+    def test_blueprints_delete_force(self):
+        for flag in ['--force', '-f']:
+            mock = MagicMock()
+            self.client.blueprints.delete = mock
+            self.invoke('cfy blueprints delete a-blueprint-id {0}'.format(
```

Comment:
this is all over the tests and at some point we should decide one way or another
but currently the `cfy` is not needed here
see https://github.com/cloudify-cosmo/cloudify-cli/blob/master/cloudify_cli/tests/cfy.py#L68